website/docs/reference/api/legacy/unleash/admin/features.md

```diff
@@ -134,11 +134,12 @@ Used to fetch details about a specific featureToggle. This is mostly provded to
 
 ## Create a new Feature Toggle {#create-a-new-feature-toggle}
 
-:::caution Deprecation notice
-This endpoint is deprecated. Please use the [project-based endpoint to create feature toggles](/reference/api/legacy/unleash/admin/features-v2.md#create-toggle) instead.
+:::caution Removal notice
+This endpoint was removed in Unleash v5. Please use the [project-based endpoint to create feature toggles](/reference/api/legacy/unleash/admin/features-v2.md#create-toggle) instead.
```

Comment:
```suggestion
:::caution Removal notice

This endpoint was removed in Unleash v5 (deprecated since v4). Please use the [project-based endpoint to create feature toggles](/reference/api/legacy/unleash/admin/features-v2.md#create-toggle) instead.

```